pkg/execute/executor.go

```diff
@@ -181,7 +181,9 @@ func NewDefaultExecutor(msg string, allowkubectl, restrictAccess bool, defaultNa
 
 // Execute executes commands and returns output
 func (e *DefaultExecutor) Execute() string {
-	args := strings.Fields(strings.TrimSpace(e.Message))
+	//Remove hyperlink if it got added automatically
```

Comment:
```suggestion
	// Remove hyperlink if it got added automatically
```
Please keep consistent format for comments

Reply:
Done